src/plotman/configuration.py

```diff
@@ -82,6 +210,9 @@ def get_dst_directories(self):
 
         return self.dst
 
+    def create_log_path(self, group, time=None):
```

Comment:
Looks like `time` shouldn't have `None` as default, as it's required by the method

Reply:
Uh, yeah...  changed my mind on this and didn't remove the default.

```suggestion
    def create_log_path(self, group, time):
```